packages/@aws-cdk/aws-stepfunctions-tasks/lib/databrew/start-job-run.ts

```diff
@@ -0,0 +1,63 @@
+import * as iam from '@aws-cdk/aws-iam';
+import * as sfn from '@aws-cdk/aws-stepfunctions';
+import { Construct } from 'constructs';
+import { integrationResourceArn, validatePatternSupported } from '../private/task-utils';
+
+/**
+ * Properties for starting a job run with StartJobRun
+ */
+export interface DataBrewStartJobRunProps extends sfn.TaskStateBaseProps {
+
+  /**
+   * DataBrew Job to run
+   */
+  readonly name: string;
+}
+
+/**
+ * Start a Job run as a Task
+ *
+ * @see https://docs.aws.amazon.com/step-functions/latest/dg/connect-databrew.html
+ */
+export class DataBrewStartJobRun extends sfn.TaskStateBase {
```

Comment:
this class should be marked `@experimental`
Rationale: it will undergo breaking changes when we leverage the Glue Databrew L2 constructs (which don't exist yet). 

It should technically take in a `IGlueDataBrewJob` or something similar at that point.